test/images/resource-consumer/common/common.go

```diff
@@ -22,7 +22,7 @@ const (
 	ConsumeMemAddress       = "/ConsumeMem"
 	BumpMetricAddress       = "/BumpMetric"
 	GetCurrentStatusAddress = "/GetCurrentStatus"
-	MetricsAddress          = "/Metrics"
+	MetricsAddress          = "/metrics"
```

Comment:
since this is a change in functionality, we should change `test/images/resource-consumer/VERSION` and push a new image too.

Reply:
It should be done now.